frontend/src/metabase/dashboard/context/context.redux.ts

```diff
@@ -173,6 +173,8 @@ export const mapDispatchToProps = {
   undoDeleteTab,
 };
 
-export const connector = connect(mapStateToProps, mapDispatchToProps);
+export const connector = connect(mapStateToProps, mapDispatchToProps, null, {
```

Comment:
pass along ref, since the component passed to this connector is now created with `forwardRef`